app/v1/models/errors/mtdErrors.scala

```diff
@@ -38,34 +44,41 @@ object TypeOfBusinessFormatError
 object AdjustedStatusFormatError
   extends MtdError("FORMAT_ADJUSTED_STATUS", "The supplied adjusted status format is invalid")
 
+object FormatAdjustmentValueError extends MtdError("FORMAT_ADJUSTMENT_VALUE", "") {
+  def withFieldName(fieldName: String): MtdError = this.copy(message = s"The format of the '$fieldName' value is invalid")
+}
+
 object SelfEmploymentIdFormatError extends MtdError("FORMAT_SELF_EMPLOYMENT_ID","The supplied self-employment ID format is invalid")
 
+object BsasIdFormatError extends  MtdError("FORMAT_BSAS_ID", "The provided Business Source Adjustable Summary ID format is invalid")
+
+
+// Rule Errors
 object SelfEmploymentIdRuleError
   extends MtdError("RULE_SELF_EMPLOYMENT_ID","A self-employment ID should be supplied for a self-employment business type")
 
 object EndBeforeStartDateError
   extends MtdError("RULE_END_DATE_BEFORE_START_DATE","The accounting period end date predates the start date")
 
-object BothExpensesError
-  extends MtdError("RULE_BOTH_EXPENSES_SUPPLIED", "Both Expenses and consolidated expenses cannot be present at the same time")
-
-object SelfEmploymentAdjusted
-  extends MtdError("RULE_SELF_EMPLOYMENT_ADJUSTED", "A self-employment business type was adjusted. Re-trigger an adjustable summary for the self-employment to correct")
+object RuleBothExpensesError
+  extends MtdError("RULE_BOTH_EXPENSES_SUPPLIED", "Both expenses and consolidated expenses cannot be present at the same time")
 
-object BsasIdFormatError extends  MtdError("FORMAT_BSAS_ID", "The provided Business Source Accounting Summary ID format is invalid")
+object RuleSelfEmploymentAdjusted
```

Comment:
```suggestion
object RuleSelfEmploymentAdjustedError
```

Reply:
We need to come up with consistent naming convention for these errors.  Half of the package is suffixed with error and half isn't